net-ftp/uftpd/uftpd-2.10.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="The no nonsense TFTP/FTP server"
+HOMEPAGE="https://github.com/troglobit/uftpd"
+SRC_URI="https://github.com/troglobit/${PN}/releases/download/v${PV}/${P}.tar.xz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="
+	>=dev-libs/libite-1.5
+	>=dev-libs/libuev-2.2"
+
+RDEPEND="
+	${DEPEND}
+	!net-misc/uftp
+	!net-ftp/atftp"
+
+src_install() {
+	emake DESTDIR="${D}" install doc_DATA=README.md
+}
```

Comment:
What would happen if you removed this whole section from the ebuild? Looks to me like it should work properly.

Reply:
You are correct. It does install README.md properly. I remove it. Thanks for pointing this out.